01-algorithms/08-algorithms-activity.md

```diff
@@ -1,209 +1,17 @@
-
-### !challenge
-
-* type: multiple-choice
-* id: e358bb4f-a3cf-412f-b491-4a3f3c7babdf
-* title: Shortest Common Supersequence
-<!-- * points: [1] (optional, the number of points for scoring as a checkpoint) -->
-<!-- * topics: [python, pandas] (Checkpoints only. optional the topics for analyzing points) -->
-
-##### !question
-
-Suppose you are given the following question:
-
-Given two strings `str1` and `str2`, find the length of the shorteset string that has both `st1` and `str2` as subsequences.
-
-**Which of the following algorithmic approaches would best suit this problem?**
-##### !end-question
-
-##### !options
-
-a| Divide & Conquer
-b| Greedy
-c| Dynamic Programming
-
-##### !end-options
-
-##### !answer
-
-c|
-
-##### !end-answer
-
-<!-- other optional sections -->
-<!-- !hint - !end-hint (markdown, hidden, students click to view) -->
-<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
-<!-- !explanation - !end-explanation (markdown, students can see after answering correctly) -->
-
-### !end-challenge
-
-<!-- ======================= END CHALLENGE ======================= -->
-<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
-<!-- Replace everything in square brackets [] and remove brackets  -->
-
-### !challenge
-
-* type: multiple-choice
-* id: 73828a6d-b5fb-40d2-ba97-a14ed729da2d
-* title: First and Last Position of Element in a Sorted Array
-<!-- * points: [1] (optional, the number of points for scoring as a checkpoint) -->
-<!-- * topics: [python, pandas] (Checkpoints only. optional the topics for analyzing points) -->
-
-##### !question
-
-Suppose you are given the following question:
-
-Given an array of integers `nums` sorted in ascending order, find the starting and ending position of a given `target` value as a tuple.
-
-If the target is not found in the array, return (-1,-1)
-
-**Which of the following algorithmic approaches would best suit this problem?**
-##### !end-question
-
-##### !options
-
-a| Divide & Conquer
-b| Greedy
-c| Dynamic Programming
-
-##### !end-options
-
-##### !answer
-
-a|
-
-##### !end-answer
-
-<!-- other optional sections -->
-<!-- !hint - !end-hint (markdown, hidden, students click to view) -->
-<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
-<!-- !explanation - !end-explanation (markdown, students can see after answering correctly) -->
-
-### !end-challenge
-
-<!-- ======================= END CHALLENGE ======================= -->
-## Solving the problem
-
-As a class, we will pick 1-2 problems to solve (as time allows). 
+### Class Livecode
 
-We will first attempt to solve the problems as a small group, then come back to talk through any sticking points and work through possible solutions as a class. 
+Follow along in the following Replit as we livecode a recursive solution with and without Dynamic Programming: [Newman-Conway Recursively](https://replit.com/@adadev/Newman-Conway-Recursive)
```

Comment:
Do we also want to add a version without the code implemented so folks who are interested can code along?

Reply:
Whoops, this version should be blank. I'll fix that!